methods.md

```diff
@@ -141,7 +150,41 @@ Use this method to verify that the pin code entered by the end-user matches the
 <tr><th>Name</th><th>Data Type</th><th>Description</th><th>Mandatory</th></tr>
 <tr><td>ServiceCredentials</td><td>ServiceCredentials</td><td>See <a href="methods.md#servicecredentials">here</a> for details</td><td>Yes</td></tr>
 <tr><td>Msisdn</td><td>String</td><td>The mobile subscription number associated with the verification code.</td><td>Yes</td></tr>
-<tr><td>VerificationCode</td><td>string</td><td>The code to be verified.</td><td>Yes</td></tr>
+<tr><td>VerificationCode</td><td>String</td><td>The code to be verified.</td><td>Yes</td></tr>
+</table>
+
+### Response types
+N/A.
+
+
+## PreAuthorize
+
+PreAuthorized payments are recurring payments where the End User have confirmed that the Pay transaction for a merchant can be executed without End User confirmation on each transaction.
```

Comment:
Since method this will (at least initially) primarily be used by users of the SMS gateway product, the term "Pay transaction" might be a bit unfamiliar. We could maybe consider using more general terms like ".. have confirmed that payment transactions for a merchant can be executed..." ? Or "Pay / Premium SMS transactions" ?

Reply:
Good point, included premium sms a couple of places.